Software/Python/easygopigo3.py

```diff
@@ -476,6 +490,27 @@ def read_encoders(self):
 
         return encoders
 
+    def read_encoders_average(self, units="cm"):
+        """
+        | Reads the encoders' position in degrees. 360 degrees represent 1 full rotation (or 360 degrees) of a wheel.
+
+        :param string units: By default it's cm, but it could also be "in" for inches. Anything else will return raw encoder average.
+
+        :returns: the average of the two wheel encoder values.
```

Comment:
I remember you making the first letter capital and yet it's still lowered. I think git is again a big pain. We need it this way
```The average of the two wheel encoder values.``` because we've got it the same way everywhere.